.eslintrc.js

```diff
@@ -12,8 +12,32 @@ module.exports = {
     '@typescript-eslint/no-unused-vars': ['error', {
       "argsIgnorePattern": "^_",
       // TODO(STENCIL-452): Investigate using eslint-plugin-react to remove the need for varsIgnorePattern
-      "varsIgnorePattern": "^(h|Fragment)$" 
+      "varsIgnorePattern": "^(h|Fragment)$"
     }],
+    /**
```

Comment:
I tried to explain/justify each of these as much as possible without copying the docs - please call out things that don't make sense/need additional details